packages/design/sass/styles.scss

```diff
@@ -166,3 +166,60 @@ iframe:focus {
   padding-bottom: 62.5%;
   height: 0;
 }
+
+.usa-fieldset {
```

Comment:
This stylesheet will override the base USWDS classes that any end-user uses, which would be unexpected. Can we use utility classes for this in the fieldset component? (Or if absolutely necessary, scope the styles to a [BEM-style modifier](https://getbem.com/introduction/) like `usa-fieldset--a2j`?)